picard/ui/mainwindow.py

```diff
@@ -198,6 +202,7 @@ def closeEvent(self, event):
         if config.setting["quit_confirmation"] and not self.show_quit_confirmation():
             event.ignore()
             return
+        config.persist['mediaplayer_volume'] = self.player.volume()
```

Comment:
When player doesn't work (no service), this returns 0, may we should only save when player actually works.